<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeriesSelection</class>
 <widget class="QScrollArea" name="SeriesSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Series Selector</string>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>277</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_dcmDir">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
